### ISSUE TITLE
Error in transit_encryption_mode attribute on AWS aws_elasticache_replication_group Resource 

### ISSUE THREAD

**Issue by webgene** [hashicorp/vscode-terraform#1766]
### Extension Version

v2.30.1

### VS Code Version

Version: 1.89.1
Commit: dc96b837cf6bb4af9cd736aa3af08cf8279f7685
Date: 2024-05-07T05:14:32.757Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Darwin arm64 23.5.0

### Operating System

macOS Sonoma Version 14.5 (23F79)

### Terraform Version

Terraform v1.8.3 on darwin_arm64

### Steps to Reproduce

1. Add transit_encryption_mode    = "preferred" in aws_elasticache_replication_group resouce.

### Expected Behavior

transit_encryption_mode Attribute added  in latest version of terraform AWS provider, hence it shouldn't show any error for this attribute.

### Actual Behavior

[
<img width="474" alt="Screenshot 2024-05-29 at 3 29 37 PM" src="https://github.com/hashicorp/vscode-terraform/assets/17011273/753b07fc-3be0-46a6-b765-8bc86c167f24">
](url)

### Terraform Configuration

```terraform
resource "aws_elasticache_replication_group" "default" {
 ......
  transit_encryption_enabled = true
  transit_encryption_mode    = "preferred"
.....
}
```

### Help Wanted

- [ ] I'm interested in contributing a fix myself

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by dbanck** [comment]
Hi @webgene, thanks for the report.

We bundle the latest provider schema available at the time of the language server and extension release. If new attributes, blocks or resources are introduced since then, we can't automatically reflect them (yet). The next language server release will bundle the updated schema and fix the problem. We apologize for the inconvenience.

In the meantime, when you run `terraform init`, we'll use the appropriate provider schema from disk.

**Comment by dbanck** [comment]
@webgene We have released a new version [2.30.2](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.30.2). This updates the schema to the latest version and should fix the validation error. The update should show up automatically in VS Code.

--- 

If you experience any other validation related bug, please let us know via a new issue.
